src/rgw/rgw_auth.cc

```diff
@@ -933,6 +934,7 @@ void rgw::auth::RemoteApplier::write_ops_log_entry(rgw_log_entry& entry) const
   if (account) {
     entry.account_id = account->id;
   }
+  entry.user = info.keystone_user;
```

Comment:
is a RemoteApplier user always a keystone user?

Reply:
no, LDAPEngine also creates RemoteAppliers

Reply:
LDAPEngine will receive an empty username, because it is not initialized with TokenEngine
https://github.com/ceph/ceph/pull/53365/files/09100fb584ce78dc1c4fd28e4e316d10341fabed#diff-2cb514dfecaff8be7cd4a8497053e815bf3a0de3d091c4472a0d134c5421dbc9R6273
same for STSEngine
https://github.com/ceph/ceph/pull/53365/files/09100fb584ce78dc1c4fd28e4e316d10341fabed#diff-2cb514dfecaff8be7cd4a8497053e815bf3a0de3d091c4472a0d134c5421dbc9R6418